### PR TITLE
Antisnake

### PR DESCRIPTION
**Pull request by Andriykooo**
- [DEMO LINK](https://Andriykooo.github.io/layout_antisnake/index.html)
- [TEST REPORT LINK](https://Andriykooo.github.io/layout_antisnake/report/html_report/)

### CODE REVIEW

**Review by amithos — APPROVED** [review]
pretty cool, but your links doesn't work (just for case if you didn't knew it)

**Comment by Andriykooo** [comment]
> pretty cool, but your links doesn't work (just for case if you didn't knew it)

Fixed

File: src/styles/blocks/antisnake.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  @for $index from 1 through 6 {
+    &__box:nth-child(#{$index})::after {
+      content: "#{$index}";
+    }
+  }
+
+  @for $index from 1 through 6 {
+    &__box:nth-child(#{$index}) {
+      background-color: rgb(255 - (($index - 1) * 51), 0, 0);
```
Comment (aionova) [inline src/styles/blocks/antisnake.scss:27]: :+1:

Comment (aionova) [inline src/styles/blocks/antisnake.scss:27]: these loops are the same, so you can do it all in 1 loop

**Review by aionova — APPROVED** [review]
Great job!

**Pull request closed without merging**